keps/sig-api-machinery/3488-cel-admission-control/README.md

```diff
@@ -1988,8 +2005,8 @@ well as the [existing list] of feature gates.
 -->
 
 - [ ] Feature gate (also fill in values in `kep.yaml`)
-  - Feature gate name:
-  - Components depending on the feature gate:
+  - Feature gate name: ValidatingAdmissionPolicy
```

Comment:
We should also update `kep.yaml`; that seems to have `CelValidatingAdmissionExtensibility` instead.

Reply:
Updated